posthog/helpers/session_recording.py

```diff
@@ -84,6 +91,7 @@ def compress_and_chunk_snapshots(events: List[Event], chunk_size=512 * 1024) ->
                     "data": chunk,
                     "compression": "gzip-base64",
                     "has_full_snapshot": has_full_snapshot,
+                    "events_summary": get_events_summary_from_snapshot_data(data_list) if index == 0 else None,
```

Comment:
Let's add a comment as to why the index == 0 case here - this is non-intuitive

Reply:
good call 👍